src/blockchain/BitcoinBlockchain.ts

```diff
@@ -67,8 +69,34 @@ export class BitcoinBlockchain {
      * @returns BitcoinFee
      */
     public async GetTxFee(): Promise<BitcoinFee> {
-        var fees = await this._prokeyBtcBlockchain.GetTxFee();
         var fee = <BitcoinFee>{};
+        if (this._prokeyBtcBlockchain._coinName === 'BTC')
```

Comment:
Using _coinName of _prokeyBtcBlockchain is not a good approach I think, while we have coinName in the class constructor.

Reply:
storing it in a new property also is not a good approach as we are taking more memory.